Code/Tools/ProjectManager/Platform/Mac/O3DE_SDK_Launcher.cpp

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Where is the cmakelists.txt file that pulls in this .cpp?

Reply:
Perhaps this should be a new tool under something like Code/Tools/SDKLauncher.  I think if it's left here, it might be easy to miss.

Reply:
I can move it there but it's also something that's needed only on Mac. Since PC/Linux don't have the concept of a bundle.

Reply:
can be in `Code/Tools/BundleLauncher` and the target can be excluded for other platforms. Basically, at the top of the CMakeLists.txt you can have a 
```
if(NOT ${PAL_PLATFORM_NAME} STREQUAL Mac)
    return()
endif()
```
Since we dont need to have different files/variants per platform, it should be ok to choose the whole think just for the platform we want.